Libs/MRML/DisplayableManager/vtkMRMLModelDisplayableManager.cxx

```diff
@@ -2061,11 +2089,22 @@ ::CreateTransformedClipper(vtkMRMLTransformNode *tnode, vtkMRMLModelNode::MeshTy
     {
     if (type == vtkMRMLModelNode::UnstructuredGridMeshType)
```

Comment:
OK, thanks for the clarification, I did not notice the vtkClipDataSet/vtkClipPolyData difference. It makes sense to keep using the mesh type to decide what filters to instantiate.